libcudacxx/docs/extended_api/synchronization_primitives/barrier.md

```diff
@@ -60,6 +61,43 @@ Programs shall ensure that this transformation would not introduce errors, for
 Under CUDA Compute Capability 6 (Pascal) or prior, an object of type
   `cuda::barrier` or `cuda::std::barrier` may not be used.
 
+## Shared memory barriers with transaction count
+
+Starting with the Hopper architecture (CUDA Compute Capability 9), a
```

Comment:
Does the layout change when compiling for sm_80 ?

Reply:
I'd phrase it as cuda::barrier<thread_scope_block> has a transaction count, period.

The APIs to actually use that, are all sm_90+. So its not that there are two different barrier types depending on whether <sm_90 or >= sm_90, there is only one, and its transaction based, but in <sm_90 this is just not observable.

Reply:
I have changed the emphasis of the paragraph to reflect your comments.